features/lorawan/lorastack/mac/LoRaMacCommand.cpp

```diff
@@ -647,9 +655,10 @@ lorawan_status_t LoRaMacCommand::add_beacon_freq_ans(uint8_t status)
 {
     lorawan_status_t ret = LORAWAN_STATUS_LENGTH_ERROR;
 
-    if (cmd_buffer_remaining() > 0) {
-        mac_cmd_buffer[mac_cmd_buf_idx++] = MOTE_MAC_PING_SLOT_CHANNEL_ANS;
+    if (cmd_buffer_remaining() > 1) {
```

Comment:
It's not obvious why this was changed from 0 to 1. CAn you add this to the commit message or is it clear, just not  to me?

Reply:
`PingSlotChannelAns` requires carries 1 byte of payload, just like e.g. `DLChannelAns` https://github.com/ARMmbed/mbed-os/pull/12204/files#diff-a4ae08541d48334074fe67979cc8f449R598